packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/edit/logs-inputs-add-edit.controller.js

```diff
@@ -45,11 +45,11 @@ class LogsInputsAddEditCtrl {
       .load()
       .then(() => {
         this.ovhSubsidiary = this.accountDetails.data.me.ovhSubsidiary;
-        return this.$q.all([this.mainOffer.load(), this.catalog.load()]);
+        return this.$q.all([this.catalog.load()]);
```

Comment:
Same here, why $q.all for only one API call?

Reply:
Fixed, thanks.